packages/parser-babel-config/README.md

```diff
@@ -43,7 +43,19 @@ This `parser` emits the following events:
   * `resource`: the parsed resource.
   * `config`: an object with a valid configuration.
 
-* `parse::babel-config::error::json`, of type `BabelConfigInvalid`
+* `parse::babel-config::error::json`, of type `BabelConfigInvalidJSON`
+  which contains the following information:
+
+  * `resource`: the parsed resource.
+  * `error`: the error emited parsing the configuration file.
+
+* `parse::babel-config::error::circular`, of type `BabelConfigInvalidJSON`
+  which contains the following information:
+
+  * `resource`: the parsed resource.
+  * `error`: the error emited parsing the configuration file.
+
+* `parse::babel-config::error::extends`, of type `BabelConfigInvalidJSON`
```

Comment:
Note: In a conversation with @molant we talked about not having the `...error::not-found` (or similar events) as most of the time just one rule will needed it, and in that case, the rule can have that logic in it.

Reply:
Can you open an issue for that?

Reply:
https://github.com/sonarwhal/sonarwhal/issues/960